data_pipelines/resources/dask_resource.py

```diff
@@ -49,6 +50,7 @@ def _provision_cluster(self, context: InitResourceContext):
             image=settings.dask_cluster_image,
```

Comment:
`image` and `task_role_policies` should not be set in the production image. Instead you should set with `cluster_arn`, `scheduler_task_definition_arn`, `worker_task_definition_arn`, and `region_name`. They should all be set via an environment variable.

With this change you won't be able to test locally without already having created the ECS-cluster, and the task definitions. You could consider trying to get around this by letting Dask try to create them itself if none of the environment variables are present. Remember to log this

Reply:
Ok, good point. Maybe we can look into this together tomorrow?

Reply:
sure!